openai_connector/models/openai_vision_session.py

```diff
@@ -0,0 +1,112 @@
+import json
+import logging
+
+from openai import OpenAI
+
+from odoo import _, api, fields, models
+from odoo.exceptions import UserError
+
+_logger = logging.getLogger(__name__)
+
+
+class OpenAISession(models.Model):
+    _name = "openai.vision.session"
+    _description = "OpenAI Vision Session"
+
+    name = fields.Char(required=True)
+    model = fields.Selection(
+        [
+            ("gpt-4o", "GPT-4o"),
+        ],
+        default="gpt-4o",
+        required=True,
+    )
+    temperature = fields.Float(default=0.7)
+    instruction = fields.Text()
+    inputs = fields.Text()
+    previous_response_id = fields.Char(string="Previous Response ID")
+    store_response = fields.Boolean(default=True)
+    web_search = fields.Boolean(string="Use Web Search", default=False)
```

Comment:
Boolean fields are False as a default.
```suggestion
    web_search = fields.Boolean(string="Use Web Search")
```

Reply:
Done